brillig_bytecode/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "brillig_bytecode"
+version = "0.1.0"
```

```suggestion
description = "The virtual machine that processes Brillig bytecode, used to introduce non-determinism to the ACVM"
version = "0.1.0"
```

We need a description for this crate in order to be able to publish to crates.io.